flybywiresim/aircraft
# feat(nxapi): Decrease TELEX API poll rate to random number 45-75.

Fixes #[issue_no]

## Summary of Changes

Changes the polling rate to FBW API from static 15 seconds to a random number between 45 & 75 (randomized) to deload the API.

## Additional context
Discord username (if different from GitHub): alepouna

## Testing instructions

1. Spawn in a runway/gate
2. Make sure you have Telex integration enabled https://docs.flybywiresim.com/fbw-a32nx/feature-guides/freetext/
3. Create a flight number in INIT A
4. Test you can receive FBW requests by sending a TELEX via MDCU AOC MENU to your own callsign.
5. You should receive this message within 45-75 seconds after sending it.

Recommend trying this with another plane as well not just self-tests.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
